CHANGELOG.md

```diff
@@ -1,40 +1,76 @@
+ * Support for `textDocument/InlayHint` in Qute templates. See [#595](https://github.com/redhat-developer/quarkus-ls/issues/595).
+ * Delay revalidation of Java files in Qute language server. See [#666](https://github.com/redhat-developer/quarkus-ls/pull/666).
+ * Delay revalidation after edit and improve cancel checking. See [#659](https://github.com/redhat-developer/quarkus-ls/pull/659).
+ * Improve memory and peformance of Qute language server. See [#654](https://github.com/redhat-developer/quarkus-ls/issues/654).
+ * Suppress undefined variable errors in certain contexts. See [#548](https://github.com/redhat-developer/quarkus-ls/issues/548).
+ * Add Rename support within Qute templates. See [#492](https://github.com/redhat-developer/quarkus-ls/issues/492).
+ * Support missing attributes for `@TemplateData` / `@RegisterForReflection`. See [#631](https://github.com/redhat-developer/quarkus-ls/pull/631).
+ * Provide `qute.native.enabled` setting. See [#629](https://github.com/redhat-developer/quarkus-ls/issues/629).
+ * Code action to add `??` at the end of the object part for `UndefinedObject`. See [#613](https://github.com/redhat-developer/quarkus-ls/issues/613).
+ * Completion for nested block section. See [#497](https://github.com/redhat-developer/quarkus-ls/issues/497).
+
+### Bug Fixes
+ * Fix NPE with data model template. See [#664](https://github.com/redhat-developer/quarkus-ls/pull/664).
+ * Template validation complains about strings containing spaces. See [#639](https://github.com/redhat-developer/quarkus-ls/issues/639).
+ * Expression indexes are wrong. See [#627](https://github.com/redhat-developer/quarkus-ls/issues/627).
+ * Simplify the resolve signature logic. See [#652](https://github.com/redhat-developer/quarkus-ls/pull/652).
+ * `QuarkusConfigPropertiesProvider` void return type check doesn't work. See [#650](https://github.com/redhat-developer/quarkus-ls/issues/650).
+ * Linked editing doesn't work if variable is used as a parameter into a section. See [#638](https://github.com/redhat-developer/quarkus-ls/pull/638).
+
+### Build
+ * Update Jenkinsfile to use Java 17. See [#669](https://github.com/redhat-developer/quarkus-ls/pull/669).
+ * Adapt to new version of m2e in JDT-LS. See [#668](https://github.com/redhat-developer/quarkus-ls/pull/668).
+ * Remove unnecessary 2019-06 release repository from target platform. See [#658](https://github.com/redhat-developer/quarkus-ls/pull/658).
+ * Updates sites are not deployed anymore to download.jboss.org. See [#623](https://github.com/redhat-developer/quarkus-ls/issues/623).
+ * P2 update sites (quarkus.jdt, qute.jdt) are not archived for new releases/tags. See [#617](https://github.com/redhat-developer/quarkus-ls/issues/617).
+ * Remove unnecessary Gson dependency in pom files.. See [#672](https://github.com/redhat-developer/quarkus-ls/pull/672).
+ * Move to LSP4J 0.13.0. See [#644](https://github.com/redhat-developer/quarkus-ls/issues/644).

```

Comment:
It looks like we use lsp4j 0.14.0 now

Reply:
Yup, this was just a typo. We just bumped it even more.